src/app/my-rewards/backers/components/Table/BackerRewardsTable.tsx

```diff
@@ -169,27 +170,15 @@ export const BackerRewardsTable = () => {
 
   return (
     <>
-      <div className="w-full overflow-x-auto bg-v3-bg-accent-80">
-        <table className="w-full min-w-[700px]">
-          <thead>
-            <BackerRewardsHeaderRow actions={actions} />
-          </thead>
-          <Suspense fallback={<div>Loading table data...</div>}>
-            <tbody>
-              {rows.map(row => (
-                <BackerRewardsDataRow key={row.id} row={row} />
-              ))}
-            </tbody>
-          </Suspense>
-        </table>
-      </div>
+      {isDesktop && <DesktopRewardsDetails rows={rows} actions={actions} />}
```

Comment:
nitpick -
to make it more readable could wrap this in a component like `RewardsDetails` and let it handle this.